### PR TITLE
Implement ThingIFAPI#getTrigger

### PR DESCRIPTION
**Pull request by yongpingchen**
- Implement ThingIFAPI#getTrigger
- Add TriggerUtils to set targetID to a Trigger instance, which is not in response from server.
- Add small tests for this API.

### CODE REVIEW

File: thingif/src/main/java/com/kii/thingif/ThingIFAPI.java [hunk 1]
```diff
@@ -1140,19 +1143,17 @@ public Trigger getTrigger(
...
-//            int schemaVersion = commandObject.optInt("schemaVersion");
-//            schema = this.getSchema(schemaName, schemaVersion);
-//            if (schema == null) {
-//                throw new UnsupportedSchemaException(schemaName, schemaVersion);
-//            }
-//        }
-//        return this.deserialize(schema, responseBody, this.target.getTypedID());
+        try {
+            Trigger trigger =  this.gson.fromJson(responseBody.toString(), Trigger.class);
+            TriggerUtils.setTargetID(trigger, this.target.getTypedID());
```
Comment (shinsuke-nara) [inline thingif/src/main/java/com/kii/thingif/ThingIFAPI.java]: @yongpingchen What do you think about adding target id to `responseBody`?

I think that using reflection is not good way in this situation.

**Comment by shinsuke-nara** [comment]
@yongpingchen Could you check my comment?

File: thingif/src/main/java/com/kii/thingif/ThingIFAPI.java (hunk 1 above)
Comment (yongpingchen) [inline thingif/src/main/java/com/kii/thingif/ThingIFAPI.java]: @shinsuke-nara use responseBody, need to convert targetID to json then convert back, which is not effective.

**Comment by shinsuke-nara** [comment]
@yongpingchen LGTM:+1:
I will merge after CI build finished.

**Pull request merged**